src/components/Textfield/textfield.scss

```diff
@@ -198,3 +198,21 @@
     }
   }
 }
+
+.textbox {
+  &.has-error {
```

Comment:
I think this is causing a problem. We can get on a call for this.

Reply:
Some CSS in console was overwriting it, have removed it